common/test/acceptance/pages/common/paging.py

```diff
@@ -63,3 +63,8 @@ def is_previous_page_button_enabled(self):
     def is_enabled(self, css):
         """Return whether the given element is not disabled."""
         return 'is-disabled' not in self.q(css=css).attrs('class')[0]
+
+    @property
+    def footer_visibility(self):
```

Comment:
Since it returns a boolean indicating whether the footer is visible or not, it should be `footer_visible`.